### PR TITLE
Merge 'noetic' into 'dashing-devel'

### PR DESCRIPTION
**Pull request by henningkayser**
Note: I would like to move to eloquent if this succeeds (already enabled in CI) and we can probably drop dashing support. Should we just switch to a `ros2` branch for simplicity? I just don't know if this is feasible with upcoming foxy support.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ Note: Bodies for meshes compute the convex hull of those meshes in order to prov
 
 Note: [shape_tools](https://github.com/ros-planning/shape_tools) package was recently merged into this package
 
+Note: `bodies::Box::corner1_` was renamed to `minCorner_` and `bodies::Box::corner2_` to `maxCorner_` in Noetic.
```
Comment (peci1) [inline README.md]: Not sure if the mention of Noetic would be useful for ROS 2 people...

File: README.md [hunk 2]
```diff
@@ -15,6 +15,10 @@ Note: Bodies for meshes compute the convex hull of those meshes in order to prov
...
+
+Note: `bodies::ConvexMesh::MeshData` was made implementation-private in Noetic and is no longer accessible from the `.h` file.
```
Comment (peci1) [inline README.md]: dtto

**Review by peci1 — COMMENTED** [review]
Looks good from my side.

File: README.md (hunk 1 above)
Comment (henningkayser) [reply README.md]: of course, thanks

**Pull request merged**